tasks/README.md

```diff
@@ -0,0 +1,96 @@
+# Tasks
+
+## Task object
+
+```
+{
+  "type":"Dev | Group",
+  "links": [
+    <link1>,
+    <link2>
+  ],
+  "endsOn":"<end date>",
+  "startedOn":"<start date>",
+  "status": "Active",
+  "ownerId":"<app owner user id>",
+  "percentCompleted":10%,
+  "dependsOn": [
+    <tasksList>
```

Comment:
What is the format of the task list?

Reply:
My bad, `dependsOn` will be an array of `<task_objects>`.